include/spirv/spir-v.xml

```diff
@@ -92,7 +92,8 @@
         <id value="39"  vendor="SirLynix" tool="Nazara ShaderLang Compiler" comment="Contact Jérôme Leclercq, https://github.com/NazaraEngine/ShaderLang"/>
         <id value="40"  vendor="NVIDIA" tool="Slang Compiler" comment="Contact Theresa Foley, tfoley@nvidia.com, https://github.com/shader-slang/slang/"/>
         <id value="41"  vendor="Zig Software Foundation" tool="Zig Compiler" comment="Contact Robin Voetter, https://github.com/Snektron"/>
-        <unused start="42" end="0xFFFF" comment="Tool ID range reservable for future use by vendors"/>
+        <id value="42"  vendor="Intel" tool="LLVM SPIR-V Backend" comment="Contact Michal Paszkowski, michal.paszkowski@intel.com, https://github.com/llvm/llvm-project/tree/main/llvm/lib/Target/SPIRV"/>
```

Comment:
I am adding `Intel` as vendor since Intel is primarily developing the SPIR-V backend now. However, this could be changed to either `Khronos` or `LLVM` (since we do get a lot of contributions from others as well and LLVM project is community driven).

Reply:
I think I'd prefer to go with something like `LLVM`, especially if we hope to get contributions from non-Khronos members.

Reply:
I agree, changed!